10up/ElasticPress
# Fix indexing button on health screen

### Description of the Change

This change fixes the indexing button on the health screen

### Alternate Designs

### Benefits

### Possible Drawbacks

### Verification Process

1. Go to index health screen
2. Click on the reindex button
3. Check if the indexing process starts

### Checklist:






### Applicable Issues

Closes: #2298 

### Changelog Entry

Fixed indexing button on health screen
